backend/hct_mis_api/api/endpoints/rdi/upload_people.py

```diff
@@ -0,0 +1,169 @@
+from typing import Any, Dict, List
+from uuid import UUID
+
+from django.db.transaction import atomic
+from django.utils import timezone
+
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import serializers, status
+from rest_framework.exceptions import ValidationError
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from hct_mis_api.api.endpoints.base import HOPEAPIBusinessAreaView
+from hct_mis_api.api.endpoints.rdi.mixin import get_photo_from_stream
+from hct_mis_api.api.endpoints.rdi.upload import BirthDateValidator, DocumentSerializer
+from hct_mis_api.api.models import Grant
+from hct_mis_api.api.utils import humanize_errors
+from hct_mis_api.apps.core.models import BusinessArea, DataCollectingType
+from hct_mis_api.apps.household.models import (
+    COLLECT_TYPES,
+    RESIDENCE_STATUS_CHOICE,
+    ROLE_PRIMARY,
+)
+from hct_mis_api.apps.program.models import Program
+from hct_mis_api.apps.registration_data.models import RegistrationDataImport
+from hct_mis_api.apps.registration_datahub.models import (
+    ImportedDocument,
+    ImportedDocumentType,
+    ImportedHousehold,
+    ImportedIndividual,
+    RegistrationDataImportDatahub,
+)
+
+
+class PeopleSerializer(serializers.ModelSerializer):
+    first_registration_date = serializers.DateTimeField(default=timezone.now)
+    last_registration_date = serializers.DateTimeField(default=timezone.now)
+    household = serializers.ReadOnlyField()
```

Comment:
household can be removed. It is an internal design

Reply:
removed